server/tests/testutils/db.py

```diff
@@ -129,7 +138,8 @@ async def transaction_conn(db: Database) -> AsyncIterator[Connection]:
 
 
 def _build_table_where_clause(table: DeclarativeMeta, **kwargs: Any) -> ClauseElement:
-    return cast(
-        ClauseElement,
-        sa.and_(*(getattr(table, key) == val for key, val in kwargs.items())),
-    )
+    if kwargs:
+        where = sa.and_(*(getattr(table, key) == val for key, val in kwargs.items()))
+    else:
+        where = sa.and_(True)
```

Comment:
It's better to append to a specified list of filters from the function arguments instead of returning these hacks.

Reply:
changed, now caller checks that kwargs is not empty